src/components/color-picker/color-picker-palette.tsx

```diff
@@ -0,0 +1,104 @@
+import { Component, h, State, Event, EventEmitter, Prop } from '@stencil/core';
+/**
+ * @private
+ */
+
+@Component({
+    tag: 'limel-color-picker-palette',
+    shadow: true,
+    styleUrl: 'color-picker-palette.scss',
+})
+export class PaletteExample {
+    /**
+     *
```

Comment:
Write appropriate doc here

Reply:
What is `searchedColor` @LucyChyzhova ? :) I don't remember. Did you add it?

Reply:
Yes, I have added it last night 😂 
We need it to send the color from the input field at the parent component to the child - picker palette